CHANGELOG.md

```diff
@@ -1,5 +1,82 @@
+* provider: changed `credentials`, `access_token` precedence so that `credentials` values in configuration take precedence over `access_token` values assigned through environment variables ([#10393](https://github.com/hashicorp/terraform-provider-google/pull/10393))
+* provider: removed redundant default scopes. The provider's default scopes when authenticating with credentials are now exclusively "https://www.googleapis.com/auth/cloud-platform" and "https://www.googleapis.com/auth/userinfo.email". ([#10374](https://github.com/hashicorp/terraform-provider-google/pull/10374))
+* pubsub: removed `path` field from `google_pubsub_subscription` ([#10424](https://github.com/hashicorp/terraform-provider-google/pull/10424))
+* resourcemanager: `org_id`, `folder_id` are unset when removed from config in `google_project` ([#10373](https://github.com/hashicorp/terraform-provider-google/pull/10373))
+* resourcemanager: added conflict between `org_id`, `folder_id` at plan time in `google_project` ([#10373](https://github.com/hashicorp/terraform-provider-google/pull/10373))
+* resourcemanager: changed the `project` field to `Required` in all `google_project_iam_*` resources ([#10394](https://github.com/hashicorp/terraform-provider-google/pull/10394))
+* runtimeconfig: removed the Runtime Configurator service from the `google` (GA) provider including `google_runtimeconfig_config`, `google_runtimeconfig_variable`, `google_runtimeconfig_config_iam_policy`, `google_runtimeconfig_config_iam_binding`, `google_runtimeconfig_config_iam_member`, `data.google_runtimeconfig_config`. They are only available in the `google-beta` provider, as the underlying service is in beta. ([#10410](https://github.com/hashicorp/terraform-provider-google/pull/10410))
+* sql: added drift detection to the following `google_sql_database_instance` fields: `activation_policy` (defaults `ALWAYS`), `availability_type` (defaults `ZONAL`), `disk_type` (defaults `PD_SSD`), `encryption_key_name` ([#10412](https://github.com/hashicorp/terraform-provider-google/pull/10412))
+* sql: changed the `database_version` field to `Required` in `google_sql_database_instance` resource ([#10398](https://github.com/hashicorp/terraform-provider-google/pull/10398))
+* sql: removed the following `google_sql_database_instance` fields: `authorized_gae_applications`, `crash_safe_replication`, `replication_type` ([#10412](https://github.com/hashicorp/terraform-provider-google/pull/10412))
+* storage: removed `bucket_policy_only` from `google_storage_bucket` ([#10397](https://github.com/hashicorp/terraform-provider-google/pull/10397))
+* storage: changed the `location` field to required in `google_storage_bucket` ([#10399](https://github.com/hashicorp/terraform-provider-google/pull/10399))
+
+VALIDATION CHANGES:
```

Comment:
Oops, none of the at least/exactly one of entries start with verbs.

Reply:
I think this is okay? Or should I add "added validation to require ..."?

Reply:
I'm really fine with it if you are, just trying to follow the rules we have written out. But we normally don't have a Validation section either, so either way :)

Reply:
Yeah, there are so many of these that they were distracting from the more important breaking changes, so we figured it would be reasonable to break them into a separate section